### PR TITLE
Enable nGraph GPU transformer

### PR DESCRIPTION
**Pull request by csullivan**
Partner PR of https://github.com/NervanaSystems/ngraph-mxnet-bridge/pull/23 for enabling ngraph nvgpu support.

### CODE REVIEW

**Comment by csullivan** [comment]
TESTNOW

File: make/config.mk [hunk 1]
```diff
@@ -124,6 +124,7 @@ USE_MKLDNN = 0
 
 USE_NGRAPH = 0
 USE_NGRAPH_IE = 0
+USE_NGRAPH_GPU = 0
```
Comment (fengleitian) [inline make/config.mk]: I didn't see this been used anywhere.

Comment (csullivan) [reply make/config.mk]: It is used by `ngraph.mk` which is included by the Makefile and is pulled in via a git submodule from the ngraph-mxnet-bridge repo. c.f. https://github.com/NervanaSystems/ngraph-mxnet-bridge/blob/master/ngraph.mk#L119

File: make/config.mk [hunk 2]
```diff
@@ -109,6 +109,9 @@ USE_MKLDNN = 0
 # USE_NGRAPH_IE - If 1, then if/when nGraph is built it will contain support for inference-engine
 #   processing.
 #
+# USE_NGRAPH_GPU - If 1, then if/when nGraph is built it will contain support for NVIDIA GPU
+#   processing. Requires USE_CUDA and USE_CUDA_PATH to be set above.
+#
```
Comment (csullivan) [inline make/config.mk]: @fengleitian's comment reminded me that we need a comment here explaining the flag, please see 
cb6dac5

Comment (mbrookhart) [reply make/config.mk]: Probably a good idea.

Comment (mbrookhart) [reply make/config.mk]: Is it possible to actually make that a hard requirement in the build system in ngraph-mxnet-bridge?

Comment (csullivan) [reply make/config.mk]: Yeah, I didn't add any check there because it is a hard requirement in ngraph though.

Comment (csullivan) [reply make/config.mk]: It will complain at build time if those criteria are not met, which is why I wasn't too worried about adding a comment about it.

**Pull request merged**